### PR TITLE
Text to Speech improvements 

### PR DESCRIPTION
**Pull request by rafaelmardojai**

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
Should I write the settings logic, although we are not going to expose in the UI for now? 

Something like `0 = Disabled` `1 = Google TTS`, and default to `1`.

Maybe we can expose it like a switch to enable/disable TTS in the preferences window, what do you say?

**Comment by rafaelmardojai** [comment]
I have added the preference to enable/disable TTS. I just need to update the window code to follow it.

File: src/tts/gtts.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    def download_voice(self, text, language, file):
+        try:
+            tts = gTTS(text, lang=language, lang_check=False)
+            tts.write_to_fp(file)
+            file.seek(0)
+
+        except Exception as exc:
+            raise TextToSpeechError(exc) from exc
+
+
```
Comment (fushinari) [inline src/tts/gtts.py:27]: Extremely minor nitpick, one newline.

Haven't tested yet, but the code looks fantastic so far!

Comment (rafaelmardojai) [reply src/tts/gtts.py:26]: Fixed!

**Comment by rafaelmardojai** [comment]
Looks like is working fine so far, but maybe I'm forgetting something. Only testing will say 😂

**Comment by fushinari** [comment]
How about also doing #142 ?

**Comment by rafaelmardojai** [comment]
> How about also doing #142 ?

I don't have time to do more today, feel free to push if you have time to do it.

**Comment by fushinari** [comment]
No worries, I'll do it tomorrow in another PR. Going to test and merge this.

**Pull request merged**